app/views/ctc/questions/dependents/does_not_qualify_ctc/edit.html.erb

```diff
@@ -14,15 +13,20 @@
     </ul>
   <% end %>
 
-  <h2><%= t("views.ctc.questions.dependents.does_not_qualify_ctc.add_anyone_else") %></h2>
-  <div class="grid-flex spacing-above-25">
-      <%= link_to Ctc::Questions::Dependents::InfoController.to_path_helper(id: current_intake.new_dependent_token), class: "button button--icon button--icon--centered button--med-wide" do %>
-        <%= image_tag("checkmark.svg", alt: "") %>
-        <%= t("general.affirmative") %>
-      <% end %>
-      <%= link_to next_path, class: "button button--icon button--icon--centered button--med-wide" do %>
-        <%= image_tag("crossmark.svg", alt: "") %>
-        <%= t("general.negative") %>
-      <% end %>
+  <div class="form-card--yes-no spacing-above-25">
```

Comment:
this change wraps the questions in the same class as the yes/no questions page so that we dont end up having different sized yes/no boxes on different pages.